server/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "atlp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon server.js"
```

Comment:
You're using Nodemon yet it's not in your dev-dependencies or dependencies. Someone without this package installed globally on the local machine can not run your project locally. Could you please add it to your dev-dependencies.

Reply:
@muhireheir 
Can you resolve the conflicts, add `node_modules` to `.gitignore`  add `nodemon` to the devDependencies?